### ISSUE TITLE
More colors

### ISSUE THREAD

**Issue by andrius** [Manjaro-Sway/manjaro-sway#253]
I love manjaro sway and use it daily. However, I am not a big fan of dark color themes for a range of reasons. Here you could find more sway and foot.ini colors: https://github.com/andrius/manjaro-sway-themes

Installation instructions for sway:
Many themes are light themes and require a light fonts and GUI themes. For example For 'ayu Light' this section should be replaced from *dark to *light
https://github.com/andrius/manjaro-sway-themes/blob/master/themes/sway/ayu%20Light/definitions#L25-L30

Most of the themes converted from the iTerm2 color schemes repository, where theirs preview is available: https://github.com/mbadolato/iTerm2-Color-Schemes.

Example of ayu Light theme:
![image](https://user-images.githubusercontent.com/26776/156070507-b49887c2-7d0b-4026-aba2-11e73d4160f2.png)

**Comment by boredland** [comment]
I think it could be a nice idea to provide a way for users to contribute themes and do what you mentioned in a semi-automated manner.

**Comment by boredland** [comment]
Imagine a light-mode/dark-mode switch on the waybar 🤭

**Comment by andrius** [comment]
Good idea, but that will turn into some waybar app and manjaro refactor. Like light/dark themes in sway/config file. From my side I can only contribute with initial input, but will try, why not.
